frontend/projects/abtesting/src/app/features/dashboard/profile/components/profile-info/profile-info.component.ts

```diff
@@ -161,6 +162,20 @@ export class ProfileInfoComponent implements OnInit, OnDestroy, AfterViewInit {
     });
   }
 
+  openDeleteUserModal(user: User) {
+    const dialogRef = this._matDialog.open(DeleteComponent, {
+      panelClass: 'delete-modal'
+    });
+
+    dialogRef.afterClosed().subscribe(result => {
```

Comment:
@YashilDepani Can we update name of result variable? may be isDeleteButtonClicked?

Reply:
code updated